integrations-and-supported-tools/altair/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+altair
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works. 
We'll be updating versions periodically.